mobile/src/main/res/values-de/strings.xml

```diff
@@ -315,6 +339,7 @@
     <string name="log_activity_action_reload">Aktualisieren</string>
     <string name="log_activity_content_description_share">Protokoll teilen</string>
     <string name="empty_log">Protokoll ist leer</string>
+    <string name="log_too_large_to_share">Das Protokoll ist zu groß, um es zu teilen. Bitte leere das Protokoll, reproduziere den Fehler und teile dann das Protokoll.</string>
```

Comment:
@mueller-ma Wir siezen den Nutzer in anderen Strings, wir sollten da konsistent sein.

Reply:
Ergibt Sinn. Ich habe es verbessert.